### PR TITLE
Adds how to set up domain whitelabel and how to view email statistics

### PR DESCRIPTION
**Pull request by adityatandon007**
Adds documentation mentioned in issue #383

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=384) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=384) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-python?pullRequest=384) it.</sub>

**Comment by adityatandon007** [comment]
@thinkingserious I couldn't understand the reason why travis is failing. I followed the contributing.md but couldn't understand why it is happening. Any help would be appreciated

**Comment by adityatandon007** [comment]
@thinkingserious I think the travis job is broken for python 2.6. Thats why travis is complaining though not sure! Let me know if you can merge this.

File: USE_CASES.md [hunk 1]
```diff
@@ -169,3 +169,17 @@ print(response.status_code)
 print(response.body)
 print(response.headers)
 ```
+
```
Comment (thinkingserious) [inline USE_CASES.md]: Please add the `Table of Contents` entries and change `sendgrid-csharp` in the URLs to `sendgrid-python`.

Thanks!

**Comment by thinkingserious** [comment]
Hi @adityatandon007,

Don't worry about the tests, our PR will be merged once you fix the issues in my review. Thanks!

With Best Regards,

Elmer

**Comment by adityatandon007** [comment]
@thinkingserious changes done as requested. Let me know if you need any other changes

File: USE_CASES.md (hunk 1 above)
Comment (adityatandon007) [reply USE_CASES.md]: fixed

**Comment by adityatandon007** [comment]
@thinkingserious waiting for your response on this

**Comment by adityatandon007** [comment]
@thinkingserious still waiting

**Comment by thinkingserious** [comment]
Hello @adityatandon007,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**